cardano_node_tests/tests/tests_plutus_v3/test_mint_build.py

```diff
@@ -247,7 +247,7 @@ class TestPlutusV3Builtins:
     batch5_overspend_scripts = ()
 
     batch6_success_scripts = plutus_common.SUCCEEDING_MINTING_BATCH6_SCRIPTS_V3
-    batch6_fail_scripts = ()
+    batch6_fail_scripts = plutus_common.FAILING_MINTING_BATCH6_SCRIPTS_V3
```

Comment:
When I test this locally, these tests make it fail (even though they are expected to fail!). For example, this case tests casing on booleans with too many branches:

```
cardano_node_tests/tests/tests_plutus_v3/test_mint_build.py::TestPlutusV3Builtins::test_plutusv3_builtins@long SUBFAILED(variant='old_batch6_prot_ok_caseBoolUnhappyMoreBranches_V3_110')                                                                                    [100%]
```

The resulting error further down in the output is completely as expected:

```
E           Script evaluation error: An error has occurred:
E           'case' over a value of a built-in type failed with
E           'case True' is out of bounds for the given number of branches: 3
```

What is going wrong here?